js/modules/Debug/Debug.js

```diff
@@ -553,6 +562,19 @@ window.GLPI.Debug = new class Debug {
         });
         sql_table_body.append(rows_to_append);
 
+        $(".copy-code").on('click', function () {
+            // copy content of code block in clipboard
+            const code = $(this).parent().find('code');
+            copyTextToClipboard(code.text());
```

Comment:
Whats is the logic behind using a custom function to write to the clipboard instead of `navigator.clipboard.writeText()` ?

From my point of view, it looks like `copyTextToClipboard` is some old code that was used a while ago before `navigator.clipboard.writeText` was implemented.
It call deprecated methods such as `document.execCommand`, maybe we are better off not using it in new code ?

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard
https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand